Simulation.java

```diff
@@ -385,6 +385,8 @@ public void run() {
 
 		try {
 
+			File outDirs = new File(Parameters.outPath);
+			outDirs.mkdirs();
```

Comment:
Would it be useful to save a copy of which `parameters.yml` version we're using to this directory too?

Reply:
I think something like this would be nice for sure -- though we cold also just add any other details to "outPrefix". (i.e., we could set outPrefix to a string describing params that have been changed from the default settings). Do you have an explicit preference?

Reply:
Let's stick with using "outPrefix" to name the file if that's easier!